app/controllers/messages_controller.rb

```diff
@@ -0,0 +1,69 @@
+class MessagesController < ApplicationController
+  before_action :set_message, only: %i[show edit update destroy]
+
+  # GET /messages or /messages.json
+  def index
+    @messages = Message.all
+  end
+
+  # GET /messages/1 or /messages/1.json
+  def show; end
+
+  # GET /messages/new
+  def new
+    @message = Message.new
+  end
+
+  # GET /messages/1/edit
+  def edit; end
+
+  # POST /messages or /messages.json
+  def create
+    @message = Message.new(message_params)
+
+    respond_to do |format|
+      if @message.save
+        format.html { redirect_to message_url(@message), notice: 'Message was successfully created.' }
+        format.json { render :show, status: :created, location: @message }
+      else
+        format.html { render :new, status: :unprocessable_entity }
+        format.json { render json: @message.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # PATCH/PUT /messages/1 or /messages/1.json
+  def update
+    respond_to do |format|
+      if @message.update(message_params)
+        format.html { redirect_to message_url(@message), notice: 'Message was successfully updated.' }
+        format.json { render :show, status: :ok, location: @message }
+      else
+        format.html { render :edit, status: :unprocessable_entity }
+        format.json { render json: @message.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # DELETE /messages/1 or /messages/1.json
+  def destroy
+    @message.destroy
+
+    respond_to do |format|
+      format.html { redirect_to messages_url, notice: 'Message was successfully destroyed.' }
+      format.json { head :no_content }
+    end
+  end
+
+  private
+
+  # Use callbacks to share common setup or constraints between actions.
+  def set_message
+    @message = Message.find(params[:id])
+  end
+
+  # Only allow a list of trusted parameters through.
+  def message_params
+    params.require(:message).permit(:greetings)
+  end
```

Comment:
- [ ] Kindly make sure to create an endpoint that selects and returns a random greeting from the database. Kindly also remember to set up the routes for that action, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/rails_react_webpack.md#:~:text=Create%20an%20API%20endpoint%20that%20selects%20a%20random%20greeting%20from%20your%20table%20(you%20will%20need%20a%20controller%20with%20an%20action%20and%20Rails%20route).) requirement.

Reply:
@ShadyShawkat thanks I will work on that as suggested in the project requirement